teacher.rb

```diff
@@ -0,0 +1,13 @@
+class Teacher < Person
+  attr_accessor :specialization
+
+  def initialize(id, age, specialization, name = 'unknown', parent_permision: true)
+    super(id, name, age, parent_permision)
```

Comment:
- [ ] [Optional] [Highly recommended] Same here, If you implement the `id` suggestion above, please remove the `id` from lines 4 and 5.